skins/oasis/modules/MenuButtonController.class.php

```diff
@@ -168,7 +168,7 @@ public function executeIndex($data) {
 	/**
 	 * @param extraReturntoquery is a string which will be urlencoded and appended to the returntoquery. eg: "action=edit".
 	 */
-	public function createLoginURL($extraReturntoquery='') {
+	private function createLoginURL( $extraReturntoquery = '' ) {
 		global $wgTitle;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgTitle" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)